### DISCUSSION TITLE
DHCP connection check for leasetime

### DISCUSSION THREAD

**Discussion by ZinnerC** [earlephilhower/arduino-pico#1759] [Q&A] [answered]
Is there a function similar to [Ethernet.maintain()](https://www.arduino.cc/reference/en/libraries/ethernet/ethernet.maintain/) which needs to be called with Wiznet5100lwIP?

**Comment by earlephilhower** [comment, marked as answer]
No, it's not needed.  The LWIP running in the Pico handles DHCP lease renewal automatically, behind the scenes.
